integration-cli/docker_cli_run_test.go

```diff
@@ -4187,6 +4187,8 @@ func (s *DockerDaemonSuite) TestRunWithUlimitAndDaemonDefault(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunStoppedLoggingDriverNoLeak(c *check.C) {
+	c.Skip("Splunk long driver isn't supported")
```

Comment:
Typo: `long` -> `log`. Also, for consistency with [other messages from above](https://github.com/resin-os/balaena/pull/25/files#diff-6d9e7586edc05102d4ad66e54659c8c5R1669), `log-driver` might be even better.